include/sharg/validators.hpp

```diff
@@ -174,7 +174,8 @@ class value_list_validator
     value_list_validator(range_type rng)
     {
         values.clear();
-        std::ranges::move(std::move(rng), std::cpp20::back_inserter(values));
+        for (auto & val : rng)
+            values.push_back(std::move(val));
```

Comment:
This is very nitpicky, but maybe worth a look?

https://quick-bench.com/q/jQCFzTYw6TuC7utVoDzZGRQmf50
On GCC, the `move_iterator` is fastest, but on Clang, it's `std::move`.

```suggestion
    std::move(rng.begin(), rng.end(), std::back_inserter(values));
```

Reply:
Interesting! Thanks.

Although performance is not critical here, if the suggestion is already there I'll try it.